generative_ai/context_caching/create_context_cache.py

```diff
@@ -54,3 +54,10 @@ def create_context_cache(project_id: str) -> str:
     # [END generativeaionvertexai_gemini_create_context_cache]
 
     return cached_content.name
+
+
+if __name__ == "__main__":
+    import google.auth
+
+    project_id = google.auth.default()[1]
```

Comment:
Please use `import os` and `PROJECT_ID=os.get...` & move them to the top portion of the file